vendor/licenses/git_submodule/sourcepawn-vscode.dep.yml

```
@@ -1,15 +1,15 @@
 ---
 name: sourcepawn-vscode
-version: 88a48c8263acd92cc877d624b3918cecfcb18d03
+version: 05d4ab604b2cf3c30c7147472cac10f046200477
 type: git_submodule
-homepage: https://github.com/Dreae/sourcepawn-vscode
-license: mit
+homepage: https://github.com/Sarrus1/sourcepawn-vscode.git
+license: other
```

I've taken a look and I think it's because the licensed gem we use isn't detecting the file as MIT for some reason so we'll need to manually override it.

```suggestion
license: mit
```